tests/BecomingTest.php

```
@@ -614,6 +743,41 @@ public function __construct(
+    ) {
+    }
+}
+
+#[Be(BecomingTestRuntimeFailingTarget::class)]
+final class BecomingTestRuntimeMiddle
+{
+    public readonly string $email;
+
+    public function __construct(
+        #[Input]
+        string $value, // phpcs:ignore SlevomatCodingStandard.Functions.UnusedParameter.UnusedParameter
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$value'. (undefined)

(UnusedFormalParameter)